cmd/createtree/main.go

```diff
@@ -151,24 +152,30 @@ func newPK(opts *createOpts) (*any.Any, error) {
 			Password: opts.pemKeyPass,
 		}
 		return ptypes.MarshalAny(pemKey)
+	case "PKCS11ConfigFile":
+		if opts.pkcs11ConfigPath == "" {
+			return nil, errors.New("empty PKCS11 config file path")
+		}
+		return ptypes.MarshalAny(&keyspb.PKCS11ConfigFile{opts.pkcs11ConfigPath})
```

Comment:
Is it reasonable to expect this config file to be accessible by whatever Trillian servers need to access it? If not, it'd be better to read the config file here and put the contents in the proto, rather than the path.